test/helpers/utils.js

```diff
@@ -13,24 +13,24 @@ window.Utils = {
         if (opts.label) {
             var $text = $el.find('.guide_label').eq(idx);
             expect($text.text()).toBe(opts.label.text);
-            expect(Number($text.attr('x'))).toBeCloseTo(opts.label.x, 0);
-            expect(Number($text.attr('y'))).toBeCloseTo(opts.label.y, 0);
+            expect(Number($text.attr('x'))).toBeCloseTo(opts.label.x, -1);
+            expect(Number($text.attr('y'))).toBeCloseTo(opts.label.y, -1);
```

Comment:
So the way Jasmine's in-built `isCloseTo` works, it checks that the expected is within `(10^-precision) / 2` of the actual. When precision is 0, that translates to 0.5 units. When precision is -1, that translates to 5 units. In these failures, the amounts were off by 0.6 - 1.2 pixels, which tripped up the "within 0.5 units" part of the matcher